dali/pipeline/data/tensor_list.h

```diff
@@ -646,24 +646,24 @@ class DLL_PUBLIC TensorList {
     return {data_.capacity()};
   }
 
-  /**
-   * @brief Set the type of the TensorList. The type needs to be set before calling
-   * the Resize function that gives the shape. Type can be changed, if the current storage
-   * is not big enough, the memory will be reallocated.
+  /** @defgroup SetTypeFunctions Type setting functions.
+   *
+   * Set the type of the current batch. It must be set before calling Resize(shape).
+   * It cannot be used to change the type after allocation happened.
+   *
+   * Resize(shape, new_type) can be used without prior set_type call or to request a different type
+   * after allocation.
+   * @{
    */
```

Comment:
The proper way:
```suggestion
  ///@{
  /**
   * Set the type of the current batch. It must be set before calling Resize(shape).
   * It cannot be used to change the type after allocation happened.
   *
   * Resize(shape, new_type) can be used without prior set_type call or to request a different type
   * after allocation.
   */
```
AFAIK `///@{ ` needs to be on its own line or in its own comment block.